react/components/utils/withDeviceHoc.tsx

```diff
@@ -8,18 +8,23 @@ export enum Device {
 }
 
 const withDevice = Component => {
-  function WithDevice(props) {
+  function WithDevice({
+    device: defaultIsDevice,
```

Comment:
```suggestion
    device: defaultDevice,
```

Reply:
Oh, didn't notice that. Thank you, i'll solve